apps/web/src/pages/quick-start/steps/Setup.tsx

```diff
@@ -146,8 +147,11 @@ const LoaderWrapper = styled.div`
   margin-top: 10px;
 `;
 
-function updateCodeSnipped(codeSnippet: string, environmentIdentifier: string) {
-  return codeSnippet.replace(APPLICATION_IDENTIFIER, environmentIdentifier);
+function updateCodeSnipped(codeSnippet: string, environmentIdentifier: string, apiKey: string) {
+  return codeSnippet
+    .replace(APPLICATION_IDENTIFIER, environmentIdentifier)
+    .replace(API_KEY, apiKey ?? '')
+    .replace(ENVIRONMENT, process.env.REACT_APP_API_URL === 'https://api.novu.co' ? '' : 'dev');
```

Comment:
We need this one in order to provide a more fluent flow for the product team so they won't need to update the API and WS URLs on the demo app.

I was not sure what indication to add here at the moment I am using the API URL, if some one have any other suggestion i would love to hear about them :)